core/platform/lf_zephyr_clock_kernel.c

```diff
@@ -65,6 +65,7 @@ int _lf_clock_now(instant_t* t) {
         last_epoch_nsec += epoch_duration_nsec;
     }
     *t = (SECOND(1)/CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC)*now_cycles + last_epoch_nsec;
+    clock_sync_apply_offset(t);
```

Comment:
Incomplete/unclear function comment.  Need to add to the function comment that the clock sync offset is added.